test/core/plugin_test.py

```diff
@@ -6,13 +6,15 @@
 def test__plugin_manager_registers_example_plugin():
     """Test that the example plugin is registered."""
     plugin_manager = get_plugin_manager()
-    # The plugin import order is non-deterministic
-    assert sorted(
-        [plugin_module.__name__ for plugin_module in plugin_manager.get_plugins()]
-    ) == [
+    # The plugin import order is non-deterministic.
+    # Use sets in case the dbt plugin (or other plugins) are
+    # already installed too.
+    assert set(
+        plugin_module.__name__ for plugin_module in plugin_manager.get_plugins()
+    ) >= {
```

Comment:
Nitpick: I tend to prefer the named functions on `set`, because it's clearer to me what operation is being used. What function does `>=` correspond to?

Reply:
_is a subset of_.

I'm happy to update it to the function name - you're right that it's clearer.